pkg/cmd/psa/psa.go

```diff
@@ -82,10 +83,34 @@ func (o *PSAOptions) Run() error {
 // checkNamespacePodSecurity collects the pod security violations for a given
 // namespace on a stricter pod security enforcement.
 func (o *PSAOptions) checkNamespacePodSecurity(ns *corev1.Namespace) (*PodSecurityViolation, error) {
-	nsCopy := ns.DeepCopy()
+	labels := getLabels(ns)
+	targetLevel := o.level
+
+	if targetLevel == "" {
+		if ns.Name == "openshift-authentication" {
+			fmt.Println("debug")
+		}
+		strictestLabels := getStrictestLabels(labels)
+		strictestLabel, ok := strictestLabels.PopAny()
+		if ok && strictestLabels.Has(psapi.EnforceLevelLabel) {
```

Comment:
```suggestion
		if strictestLabel == psapi.EnforceLevelLabel || strictestLabels.Has(psapi.EnforceLevelLabel) {
```

Reply:
Yeah, lol. I just moved the popping down.